src/MapViewController.swift

```diff
@@ -76,7 +76,8 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
   var currentRouteMarker: TGMapMarkerId?
   open var shouldFollowCurrentLocation = false
   open var findMeButton = UIButton(type: .custom)
-  open var currentAnnotations: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
+  open var annotationToMarkerId: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
```

Comment:
I like the idea of an object to encapsulate both items but I will still have the original problem of only having either the marker id or the pelias annotation. Going to go with the dictionary extension for now and will keep an eye on the new TG marker work for an opportunity to refactor.